triqs/utility/timer_chrono.hpp

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
My reasoning for separating the timer from the other time code is that the `timer_chrono.hpp` uses a high resolution clock which we might want to change to a different type in the future.

Reply:
Our point was that triqs does not need both a highres and a lowres timer but rather just one timer class.

Reply:
Ok! Good.

I do not know enough about the dependencies of the previous timer to dare to remove it, while at the same time I need the hi-res timer to get ok numbers on the very fast measurements.

By adding a hi-res one rather than modifying the existing one,  I am not breaking any old code. Is that ok?